jerry-debugger/jerry-client-ws.html

```diff
@@ -1207,14 +1267,21 @@
   {
     case "b":
     case "break":
-      debuggerObj.setBreakpoint(args[2]);
+      debuggerObj.setBreakpoint(args[2], false);
```

Comment:
I think "undefined" is same as "false", so we might not even need to pass anything here.